File: JavaAssignment-1/IncBy7.java
import java.util.Scanner;
class IncBy7
{
public static void main(String[] args)
{
int num;
System.out.println("Enter any integer");
Scanner sc=new Scanner(System.in);
num=sc.nextInt();
System.out.println("Entered Integer is:"+num);
for(int i=0;i<7;i++)
{
++num;
}
System.out.println("Incremented value:"+num);
}
}

File: C++ Assignments/Assignment 1/inc_by_7.cpp
#include<iostream>
using namespace std;
int main()
{
    int num;
    cout<< "Enter an integer \n";
    cin>>num;
    for(int count=0;count<7;count++) 
    {
    num++;
    }
    cout<<"Integer when incremented by 7:\n"<<num;
    return 0;
}